docs/index.rst

```diff
@@ -23,7 +23,9 @@ Mu: A Python Code Editor
 Quickstart
 ----------
 
-Mu works with Python 3.6 or above.
+Mu works with Python 3.6 or above. You need to have Python installed
```

Comment:
Mu currently works with Python 3.5 to 3.8.

Reply:
Doh :facepalm: